apps/api/src/server/app/case-officer/appeal-formatter.js

```diff
@@ -56,8 +53,13 @@ const appealFormatter = {
 			QuestionnaireDueDate: appeal.startedAt ? formatDate(add2Weeks(appeal.startedAt)) : ''
 		};
 	},
-	formatAppealForParallelStates: function (appeal) {
+	/**
+	 * @param {{ appealStatus: object; id: any; reference: any; address: object; startedAt: any; }} appeal
+	 *  @returns {object}
```

Comment:
I don't know if you want to define specifically what keys we can expect in the response?